example/lib/pages/settings/settings-page.dart

```diff
@@ -0,0 +1,299 @@
+import 'package:example/pages/settings/settings-page-state.dart';
+import 'package:floater/floater.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+class SettingsPage extends StatefulWidgetBase<SettingsPageState> {
+  SettingsPage() : super(() => SettingsPageState());
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: _settingsAppBar(
+        goBack: this.state.goBack,
+      ),
+      body: Padding(
+        padding: const EdgeInsets.all(20.0),
+        child: SingleChildScrollView(
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.start,
+            children: [
+              _displayInfo(),
+              _themeInfo(
+                themeNotifier: this.state.themeNotifier,
+                isDarkMode: this.state.isDarkMode,
+                toggleTheme: this.state.toggleTheme,
+              ),
+              _aboutInfo(
+                userName: this.state.userName,
+              ),
+              _statusInfo(),
+              _privacyPolicy(),
+              _logout(
+                logout: this.state.logout,
+              ),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class _displayInfo extends StatelessWidget {
+  const _displayInfo({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        SizedBox(
+          height: 20,
+        ),
+        const Row(
+          children: [
+            Text(
+              "Display",
+              style: TextStyle(
+                fontSize: 21,
+                fontWeight: FontWeight.w500,
+              ),
+            ),
+          ],
+        ),
+        const SizedBox(
+          height: 20,
+        ),
+      ],
+    );
+  }
+}
+
+class _themeInfo extends StatelessWidget {
```

Comment:
isn't theme under display settings?

Reply:
I just added things for filling the space, primarily i need theme and logout only in that page.

Reply:
ya but doesn't theme belong to display

Reply:
why theme widget is still outside display widget. you should have a display widget and theme should be part of it